data/result-exporter/cne-exporter-commons/src/main/java/com/powsybl/openrao/data/cneexportercommons/CneUtil.java

```diff
@@ -76,4 +92,84 @@ public static String generateUUID() {
         usedUniqueIds.add(uuidString);
         return uuidString;
     }
+
+    public static RaoParameters getRaoParametersFromProperties(Properties properties) {
```

Comment:
this retrofitting is useless. we only need three attributes in RaoParameters : relativePositiveMargins (bool), loopFlows (bool), mnecAcceptableMarginDiminution (double). See CneHelper constructor.